docs/spec.md

```diff
@@ -4770,22 +4770,27 @@ The following diagram shows how elements in `result` are computed from
 
 More formally:
 
- * `selected_values = reduce_window_without_init(...)` with the following inputs:
-   * `inputs` $=$ [ `operand` ].
-   * `window_dimensions`, `window_strides`, and `padding` which are used as is.
-   * `base_dilations` $=$ `windows_dilations` $=$ `[1, ..., 1]`.
+ * `selected_values[index] = reduce_without_init(window, axes(operand), body)` where:
```

Comment:
I think the current problem is not caused by the formalism using `reduce_window` (which I think fits into this formalism nicely, allowing to avoid speccing how to compute windows). The problem is that `selected_values[index]` needs to be populated by something for each `index`, and it's unclear what that something should be if the entire window consists of padding values. This formulation doesn't address this problem either.